src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Review.cshtml

```diff
@@ -1,6 +1,49 @@
-﻿@page
+﻿@page "{id?}"
 @model APIViewWeb.Pages.Assemblies.ReviewModel
 @{
     ViewData["Title"] = "Review";
 }
-<pre>@Html.Raw(Model.AssemblyModel)</pre>
\ No newline at end of file
+
+<pre>@Html.Raw(Model.AssemblyModel)</pre>
+
+<div>
+    <h3>Comments</h3>
+    <br />
+
+    @if (Model.Comments != null && Model.Comments.Any())
+    {
+        @foreach (var comment in Model.Comments)
+        {
+            <div class="commentBox">
+                <b>Element: </b><a href="#@comment.ElementId">@comment.ElementId</a>
+                <br /><br />
```

Comment:
Don't use </br> for layout. Prefer bootstrap types. We can handle this later though.